browser-entry.js

```diff
@@ -71,8 +71,11 @@ process.listenerCount = function (name) {
 
 process.on = function (e, fn) {
   if (e === 'uncaughtException') {
-    global.onerror = function (err, url, line) {
-      fn(new Error(err + ' (' + url + ':' + line + ')'));
+    global.onerror = function (err, url, line, colno, error) {
+      if (!error) {
+        error = new Error(err + ' (' + url + ':' + line + ')');
+      }
+      fn(error);
```

Comment:
Aha! I'd filed #5106 before seeing this PR. Great. I'll close that one since this once came first.

[Feature] What do you think about including `':' + col` too, per https://github.com/mochajs/mocha/pull/5107/files#diff-c282db01f80ee266af40ae81278d9d99177c763ae242fdeb38ee33c603598a2fR75 ?

Request adding it in as a new nicety, unless you know of a reason not to.